sdk/ai/ai-inference-rest/samples/v1-beta/javascript/telemetry.js

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Demonstrates how to get instrumentation by open telemetry.
+ *
+ * @summary get instrumentation by open telemetry.
+ */
+
+const { diag, DiagConsoleLogger, DiagLogLevel } = require("@opentelemetry/api");
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+const { createAzureSdkInstrumentation } = require("@azure/opentelemetry-instrumentation-azure-sdk");
+const { ConsoleSpanExporter, NodeTracerProvider, SimpleSpanProcessor } = require("@opentelemetry/sdk-trace-node");
+
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.VERBOSE);
+
+const provider = new NodeTracerProvider();
+provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+provider.register();
+
+// register Azure SDK Instrumentation.
+// ************** IMPORTANT SETUP STEP ***************
+// this must be done before loading @azure/core-tracing
+registerInstrumentations({
+  instrumentations: [createAzureSdkInstrumentation()],
+});
+
+
+const ModelClient = require("@azure-rest/ai-inference").default;
+const { isUnexpected } = require("@azure-rest/ai-inference");
+const { AzureKeyCredential } = require("@azure/core-auth");
+const { createTracingClient } = require("@azure/core-tracing");
+
+const tracingClient = createTracingClient({
```

Comment:
wait, end-users are not expected to use this API, it's for our internal use only. 

Please use OpenTelemtry API for it - https://opentelemetry.io/docs/languages/js/instrumentation/#create-spans

Reply:
Done